src/store/accounts/account-activity.ts

```diff
@@ -110,3 +111,20 @@ export const cleanupLocalTxs = atom(null, (get, set) => {
     set(anAtom, newLocalTxs);
   }
 });
+
+export const removeLocalSubmittedTxById = atom(null, (get, set, txIdToRemove: string) => {
```

Comment:
Can't we just have a hook that return a function instead of an atom ?

Reply:
There is: https://github.com/hirosystems/stacks-wallet-web/blob/99a619bdd471bdd8b4e94c1cd5011520c883ce0c/src/store/accounts/account-activity.hooks.ts#L72-L74